src/hotspot/share/gc/shared/gcArguments.cpp

```diff
@@ -172,3 +172,9 @@ void GCArguments::initialize_heap_flags_and_sizes() {
 
   DEBUG_ONLY(assert_flags();)
 }
+
+size_t GCArguments::heap_virtual_to_physical_ratio() {
+  // Used by heap size heuristics to determine max amount
+  // of address space to use for the heap.
```

Comment:
Could this explanation moved to the hpp file? Then some IDEs will automatically pick it up. I'd expect an explanation of the value 1 here instead.

Reply:
Ok, I'll move the comment.